### ISSUE TITLE
Admin menu seems scrambled / parts missing [2.1.0rc2]

### ISSUE THREAD

**Issue by Asitis** [magento/magento2#4948]
## Steps to reproduce
1. Clone Magento 2.1.0rc2 (`git clone git@github.com:magento/magento2.git`)
2. Install via Composer
3. Login to admin  Stores > Configuration
## Expected result

See all the configuration options
## Actual result

Only 'Search Engine Robots' appears on the Design tab. 
The 'Web' tab is completely empty.
The 'General' tab everything is set to 'System value' even though this would be the top-scope to change.
And more of these 'scrambled' options, a lot of options are missing, some are misplaced, others are 'disabled'.
There are no console/JS errors or other related errors in the logs.

![schermafbeelding 2016-06-10 om 09 26 43](https://cloud.githubusercontent.com/assets/9464641/15957316/a001415e-2eee-11e6-9139-ada11a8b17e3.png)
![schermafbeelding 2016-06-10 om 09 36 17](https://cloud.githubusercontent.com/assets/9464641/15957343/d08b89a6-2eee-11e6-927b-e4e5726ae97c.png)
![schermafbeelding 2016-06-10 om 09 38 34](https://cloud.githubusercontent.com/assets/9464641/15957387/247f680c-2eef-11e6-9382-fd71fc5555f5.png)

Edit: Did a new re-install to see what would happen. This time the "Web" tab isn't empty, but everything else is still the same.

**Comment by Asitis** [comment]
I found this topic: https://github.com/magento/magento2/issues/4754
Apparently the theme options moved to Content > Design:Configuration in the backend. And I guess it would be possible that order menu-options got shuffled around this release.

TL;DR: Sorry, seems to be a non-issue, all the options are there, but in different locations as before.

Update: Saw the same issue again where the "Web" tab was completely empty. It appears this was caused by a faulty import of some database tables referring ID's that didn't exist in this installation. See this feature request for a possible solution: https://github.com/magento/magento2/issues/4956
